lib/game_box_web/components/typography.ex

```diff
@@ -0,0 +1,176 @@
+defmodule GameBoxWeb.Typography do
```

Comment:
Are these files generated? If so, how are they generated? Is this something that happens automatically at compilation based on the tailwind config?

Reply:
This is not a generated file. It is based off of the structure of petal.build components, which is very similar to the structure of the new Phoenix core components. I tend to start with this structure of a file as a base, and then I customize it and keep all my typography in this file. You'll notice a similar structure (and reliance on the same component helpers) as I build out containers, dropdowns, etc.